jinja2/base.html

```diff
@@ -253,20 +269,27 @@ <h2 class="footer-title">Mozilla</h2>
 
   <!-- site js -->
   {% block site_js %}
-    <!--[if lte IE 8]>{% javascript 'selectivizr' %}<![endif]-->
-
-    {{ providers_media_js() }}
-    <script type="text/javascript" src="{{ statici18n(request.LANGUAGE_CODE) }}"></script>
-    {% javascript 'main' %}
-    <script>
-      if(window.mdn && mdn.analytics) mdn.analytics.trackOutboundLinks();
-    </script>
-    {% for script in scripts %}
-      {% javascript script %}
-    {% endfor %}
+    <![if !lte IE 9]>
```

Comment:
The conditional comments are inside the django block here and outside below. I think we should be consistent.